### ISSUE TITLE
go install tip is failing 

### ISSUE THREAD

**Issue by Shaked** [moovweb/gvm#125]
Hey, 

I am trying to install `go tip` and it seems like something is going wrong. 
```
Shaked@mac ~ $ gvm install tip
Downloading Go source...
^@^@Updating Go source...
ERROR: Unrecognized Go version
```

I am using Mac 10.10.2. 
I have reinstalled `gvm` after using ```gvm implode```
Gvm & Go information: 
```
Shaked@mac ~ $ gvm list

gvm gos (installed)

=> go1.4
   system

Shaked@mac ~ $ gvm version
Go Version Manager v1.0.22 installed at /Users/Shaked/.gvm
Shaked@mac ~ $
```

Any idea? 

Thanks,

**Comment by corny** [comment]
Any solutions for this issue?

**Comment by ernado** [comment]
:+1:

**Comment by corny** [comment]
`gvm install master` works for me

**Comment by ernado** [comment]
@corny, is it in `listall`?
I wanted use go1.5 via gvm, but do not understand how

**Comment by corny** [comment]
No, it is not in `listall`. I installed current development version of Go with the following commands:
```
gvm install go1.4.2
gvm use go1.4.2 --default
GOROOT_BOOTSTRAP=~/.gvm/gos/go1.4.2 gvm install master
gvm use master --default
```

**Comment by Zilev** [comment]
I also have the same problem, after apply the suggestions didn't work, I have:
´´´
gvm version
Go Version Manager v1.0.22 installed at /path/.gvm
´´´
´´´
gvm install springboot
Updating Go source...
ERROR: Unrecognized Go version

´´´
please someone can help me?

**Comment by AlexAznar** [comment]
gvm install go1.4.2
gvm use go1.4.2

it's works for me
